Makefile

```diff
@@ -97,7 +97,13 @@ cleanup-ack-req: verify-cluster-variables
 	yq e '.cluster.region=env(CLUSTER_REGION)' -i tests/e2e/utils/ack_sm_controller_bootstrap/config.yaml
 	cd tests/e2e && PYTHONPATH=.. python3.8 utils/ack_sm_controller_bootstrap/cleanup_sm_controller_req.py
 
-
-deploy-kf-vanilla: bootstrap-ack
-	while ! kustomize build deployments/vanilla | kubectl apply -f -; do echo "Retrying to apply resources"; sleep 30; done
-        
+deploy-kubeflow: bootstrap-ack
+	$(eval DEPLOYMENT_OPTION:=vanilla)
+	$(eval INSTALLATION_OPTION:=kustomize)
+	$(eval AWS_TELEMETRY_OPTION:=enable)
+	cd tests/e2e && PYTHONPATH=.. python3.8 utils/kubeflow_installation.py --deployment_option $(DEPLOYMENT_OPTION) --installation_option $(INSTALLATION_OPTION) --aws_telemetry_option $(AWS_TELEMETRY_OPTION)
+
+delete-kubeflow:
+	$(eval DEPLOYMENT_OPTION:=vanilla)
+	$(eval INSTALLATION_OPTION:=kustomize)
```

Comment:
nit: doesnt this need telemetry option?

Reply:
No 
`delete-kubeflow` will always delete telemetry